apps/central-scan/frontend/src/components/toggle_test_mode_button.tsx

```diff
@@ -1,98 +1,91 @@
-import { Button, Modal, P } from '@votingworks/ui';
-import React, { useCallback, useRef, useState } from 'react';
-import { useHistory } from 'react-router-dom';
-import { setTestMode } from '../api';
-
-export interface Props {
-  canUnconfigure: boolean;
-  isTestMode: boolean;
-}
+import { Button, Modal, P, SegmentedButton } from '@votingworks/ui';
+import React, { useState } from 'react';
+import { iter } from '@votingworks/basics';
+import { getStatus, getTestMode, setTestMode } from '../api';
 
 /**
  * Presents a button to toggle between test & live modes with a confirmation.
  */
-export function ToggleTestModeButton({
-  canUnconfigure,
-  isTestMode,
-}: Props): JSX.Element {
-  const history = useHistory();
+export function ToggleTestModeButton(): JSX.Element | null {
```

Comment:
new experience

https://github.com/user-attachments/assets/6d44d6d9-fdf5-4823-bf9f-616cf2a91020

Reply:
old experience

https://github.com/user-attachments/assets/29a94645-1b2f-4e46-99be-8b4fd6753e78

Reply:
I'm not sure how people feel about the pattern of having the warning below the toggle, but I like it. It's what we currently do in VxMark. Even though switching from test to official is "safe", it's still possible that there may be test data you want to preserve or at least be aware you are deleting. It puts the information up front rather than forcing you through a modal.

Reply:
The double warning feels a tad excessive. I think confirm modal is our more common pattern so I'd stick with that and remove the inline warning. Also in the disabled state there are two inline warnings which is a bit overwhelming.

I'd suggest a pattern of:
- Add an inline explanation for unclear disabled buttons
- Use a confirm modal for dangerous actions

If you think switching from test to official is dangerous, then I think it warrants a confirmation modal.